app/routes.py

```diff
@@ -0,0 +1,291 @@
+from flask import Blueprint, jsonify, request, make_response
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from datetime import datetime
+import datetime
+
+#BLUEPRINTS 
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+customer_bp = Blueprint("customer_bp", __name__,url_prefix="/customers")
+rental_bp = Blueprint("rental_bp", __name__,url_prefix="/rentals")
+
+#VIDEO ROUTES
+
+@videos_bp.route("", methods=["GET", "POST"], strict_slashes=False)
+def create_videos():
+    if request.method == "POST":
+        request_body = request.get_json()
+        input_check = validate_video_input(request_body)
```

Comment:
Great use of a helper function.  

Just a note, that doing this check and the following if-statement is a bit awkward.  Another way to do this is with a decorator which can dry this up a bit. Not a problem for this project, but something to note for future learning.